domains/runtime/evm/src/lib.rs

```diff
@@ -1560,7 +1560,7 @@ impl_runtime_apis! {
     impl fp_rpc::ConvertTransactionRuntimeApi<Block> for Runtime {
         fn convert_transaction(transaction: EthereumTransaction) -> <Block as BlockT>::Extrinsic {
             UncheckedExtrinsic::new_unsigned(
-                pallet_ethereum::Call::<Runtime>::transact { transaction }.into(),
+                pallet_ethereum::Call::transact { transaction }.into(),
```

Comment:
Interesting that Rust does not complain about the Generic here

Reply:
It's already part of the `RuntimeCall` definition here:
https://github.com/autonomys/subspace/blob/eabc2ac7b133e6ad41a022f92161ad468349148d/domains/runtime/evm/src/lib.rs#L135-L137